pkg/engine/variables/operator/allin_test.go

```diff
@@ -0,0 +1,51 @@
+package operator
+
+import (
+	"testing"
+
+	"github.com/go-logr/logr"
+)
+
+func Test_allKeyExistsInArray1(t *testing.T) {
+	type args struct {
+		key   string
+		value interface{}
+		log   logr.Logger
+	}
+	tests := []struct {
+		name            string
+		args            args
+		wantInvalidType bool
+		wantKeyExists   bool
+	}{
+		{
+			name: "key exists in array",
+			args: args{
+				key:   "kube-system",
+				value: []interface{}{"kyverno", "kube-*"},
+			},
+			wantInvalidType: false,
+			wantKeyExists:   true,
+		},
+		{
+			name: "key not exists in array",
+			args: args{
+				key:   "default",
+				value: []interface{}{"kyverno", "kube-*"},
+			},
+			wantInvalidType: false,
+			wantKeyExists:   false,
+		},
```

Comment:
Shouldn't we test array/array and array/string too ?

Reply:
Then it is a new func. I will add the func for it.

Reply:
Why a new func ? This is not new test cases ?

Reply:
I add the new ut, PTAL.